core/src/wallet/cosmos/CosmosLikeAccount.cpp

```diff
@@ -626,20 +627,22 @@ namespace ledger {
 
                 std::string CosmosLikeAccount::getSequence()
                 {
-                    if (!_accData) {
+                    // Assuming _accountData is only valid if accountNumber has been fetched
+                    if (!_accountData || _accountData->accountNumber.empty()) {
```

Comment:
You mean `_accountData->sequence` here I think

Reply:
Oh nevermind I think I got the logic wrong: actually you test against `accountNumber` to check that the account is synchronized.

Reply:
Not really, the `sequence` can be empty (maybe if the account never had any transaction), but I assumed that the accountNumber never is. It may be a wrong assumption now that I think about it, but I need to check that

Reply:
Turns out I was wrong.